object/lock.go

```diff
@@ -41,7 +43,9 @@ func (x Lock) ReadMembers(buf []oid.ID) {
 	var i int
 
 	(*v2object.Lock)(&x).IterateMembers(func(idV2 refs.ObjectID) {
-		_ = buf[i].ReadFromV2(idV2)
+		if err := buf[i].ReadFromV2(idV2); err != nil {
+			panic(fmt.Errorf("invalid member #%d: %w", i, err))
```

Comment:
`panic`? In `ReadMembers`?

Reply:
there is no external usage that could lead to the failure, so panic. Silence is bad, nobody checks for zeroness, and method returns void